### ISSUE TITLE
What is the difference between a value specification and a measurement datum ?

### ISSUE THREAD

**Issue by cfrancois7** [obi-ontology/obi#833]
Dear everyone,

I don't clearly understand the difference between the two concepts and when use one or the other.
For instance, I want the total measurement of a flow that outflows a process (e.g. the total quantity of blood that ouflowed the mechanical pump during an surgery act). How I represented the datum?

Need I used this form:
total flow measurement ro:is about blood outflow;
total flow measurement iao:has measurement value xsd:float;
                                        iao:has measurement unit label;

total flow measurement obi:has_value_specification _:value;
_:value iao:has measurement unit label;
_:value obi:has specified value xsd:float.

I don't clearly understand when I should use on or the other?
I understand the use of specified value when it is about a protocol but not when it is about a measurement. Please help to keep my hair on my head.

**Comment by jamesaoverton** [comment]
We're trying to move from your first example to your second example. This change is motivated by the desire to talk about more than just measurements, e.g. predictions, settings, and simulations. These are *about* very different things: a measurement datum is about the mass of this particular mouse as measured by a particular assay; a predicted data item is about something in the future or a plan for the future; a setting value might be about a plan for the state of a device; I'm not sure what a data point from a simulation would be about. But what they all have in common is that they specify a value such as "100 g". We're trying to factor out the common values and put them all in one branch OBI, then reuse them in the other parts of OBI.

This is work in progress. If you're only dealing with measurements, you can stick to your first example, at least until we figure things out. We appreciate your question, and wlecome your input with this design process.

There's a longer (older) discussion here: https://github.com/obi-ontology/obi-legacy-svn/blob/master/trunk/src/examples/development/data-prototype.pdf

**Comment by jamesaoverton** [comment]
Here's a link to the latest design discussion: https://docs.google.com/document/d/10Mt3zb73iGhM6j1pGbP7rGE4CPSKiQBGG_TM9hJcczU/edit

I hope I've answered your question. I'll close this issue now, but you are welcome to ask further questions.

**Comment by cfrancois7** [comment]
Thank you for your link.

My question is about the last document you shared me.
I'm thinking why it is not possible to link the value specification directly to john without to use the "measurement datum" layer. I don't really understand in this case the added value of this layer.

For example, if we have several value of the john's mass: one real measure and another one that is the diet objective.
Why it is not possible to represent the data in this way:

```
: value specification of John's mass at T
type: mass value specification
type: measurement datum
is about: John's mass at T
has specified value > xsd:real: 70.0
has measurement unit label > link: kilogram [UO:0000009]

: value specification of John's mass at T
type: mass value specification
type: objective value
is about: John's mass at T
has specified value > xsd:real: 68.0
has measurement unit label > link: kilogram [UO:0000009]
```

I don't really get the inconvenient of a such representation regarding the OBI one, but in this case it is much more easier to query every (or only one type) value about John's mass. At the same times, the specification is statement about the John's mass and a measurement datum in the first case, and in the second one, it is still a statement but an objective value, e.g. regarding a diet or something else. We know it is a mass value but it is also something else : a measurement value, an estimated value, or an objective value, etc.

If the mass measurement datum has several value with different T, why not created several mass value specification link with different 'has time stamp' > 'time measurement datum'.

**Comment by jamesaoverton** [comment]
Your example of asserting multiple RDF types for these subjects could work from a technical perspective, but I think that the three classes you use ('measurement datum', 'objective value', and 'mass value specification') are disjoint from an ontological perspective.

It's not clear to me that both the subjects in your example are 'is about' the same thing, but if they are then the query you want to do would not be much more complex if the value specifications are separate.

**Comment by cfrancois7** [comment]
We can suppose a diet (a process) with a plan specification and a intended value (sub-class of value specification) (see image).

My question is: if is it really necessary to separate the value specification from the measurement datum or an intended value if the types of what is specified is well defined (e.g. a measurement datum or a predicted value, or an intended value etc.)?
In fact, I just question the possibility of a simplification of the representation. It is just I exchange in my field with some people not really in touch with the ontological representation of data and need to justify each layer, and I blocked on this one.

<img width="1032" alt="capture d ecran 2017-06-14 a 19 27 31" src="https://user-images.githubusercontent.com/11422336/27145870-8ce2860c-5137-11e7-9338-2f5ca020cf8f.png">

**Comment by jamesaoverton** [comment]
The goal has been to "factor out" the part that is shared among measurements, predictions, settings, simulated data, etc. into the value specification branch. This means that the value specification branch can be reused without being duplicated across different parts of OBI. It also means that queries are consistent, no matter whether the value specification is part of a measurement, prediction, setting, etc., which is not true of your diagram. I consider that consistency more important than saving one node in the RDF representation.

OBI is a community effort, and I'm just one developer. I'd encourage other developers to chime in if they have another perspective. Here's my perspective on what OBI is good at and how I use it:

OBI's primary goal (in my opinion) is to support data integration. Every scientific investigation and every scientific database collects and organizes information in a different way for its own specific purpose, and that's a good thing. OBI does not dictate the data representation for a project. When working with project data, I use spreadsheets and relational databases with project-specific schemas far more than I use RDF. I use ontology terms in these tables, and the columns have semantics, but it's not an RDF representation.

RDF's strength is flexibility, which makes it excellent for integrating data sets from different sources that have different representational goals. When it's time to share project data, I translate it from tables to triples modelled with OBI. But the flexibility of RDF makes it inefficient and inconvenient for many tasks. So when I start working with some RDF data, I usually run a SPARQL query to get the parts I care about into table, where I can work with it more conveniently.

The tables I'm talking about have columns for "specified value" and "unit" (if it varies) but they don't have a separate column for "value specification". The value specification node is only part of the RDF representation.

In short, OBI's goal (IMO) is data integration for any biomedical investigation, not a convenient data representation for some particular investigation. Projects have their own convenient data representations, which can be translated to RDF with OBI. So I put more weight on ontological accuracy and consistency than on the number of nodes in the RDF representation.

**Comment by cfrancois7** [comment]
Thank you for your response and your experience.

It is exactly in this way I re-used OBI. The fact is with the value specification representation, I get a similar way to query the data from measurement than model.

The quality <-> the variable
the measurement <-> the expected value
the specification value <-> the specification value.
